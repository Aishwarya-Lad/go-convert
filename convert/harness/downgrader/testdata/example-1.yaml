stages:
  - name: job1
    spec:
      platform:
        arch: amd64
        os: linux
      runtime:
        spec: {}
        type: cloud
      steps:
        - name: echo
          spec:
            envs:
              MESSAGE: world
            uses: actions/hello-world-javascript-action@main
            with:
              who-to-greet: $MESSAGE
          type: action
    type: ci
version: 1
