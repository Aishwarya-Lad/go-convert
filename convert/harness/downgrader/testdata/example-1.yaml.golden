pipeline:
  identifier: default
  name: default
  orgIdentifier: default
  projectIdentifier: default
  properties:
    ci:
      codebase:
        build: <+input>
  stages:
  - stage:
      identifier: job1
      name: job1
      spec:
        cloneCodebase: true
        execution:
          steps:
          - step:
              identifier: echo
              name: echo
              spec:
                env:
                  MESSAGE: world
                uses: actions/hello-world-javascript-action@main
                with:
                  who-to-greet: $MESSAGE
              timeout: ""
              type: Action
        platform:
          arch: Amd64
          os: Linux
        runtime:
          spec: {}
          type: Cloud
      type: CI
